reporting_app/__init__.py

```diff
@@ -12,7 +14,20 @@
 app.secret_key = cfg['key'].encode()
 login_manager = flask_login.LoginManager()
 login_manager.init_app(app)
-version = open(join(dirname(dirname(__file__)), 'version.txt')).read()
+version = open(join(dirname(dirname(__file__)), 'version.txt')).read().strip()
+
+
+def get_git_revision_short_hash():
+    try:
+        return subprocess.check_output(['git', 'rev-parse', '--short', 'HEAD']).decode('utf-8').strip()
+    except (subprocess.CalledProcessError, FileNotFoundError):
+        return None
+
+
+git_commit = get_git_revision_short_hash()
+if git_commit:
+    # This should not appear once deployed in production.
+    version = version + '-' + git_commit
```

Comment:
This bit adds the git commit to the version displayed on the reporting app.
That allows the evidence to link back to a code commit

Reply:
I don't like it, but don't see any alternative without changing the deploy process. See [here](https://stackoverflow.com/questions/3998869/show-the-latest-commit-value-within-a-web-application) for a discussion on including commit info in a web app